api/proto/teleport/userloginstate/v1/userloginstate.proto

```diff
@@ -48,4 +48,16 @@ message Spec {
   // original_traits are the user traits that are part of the user's static definition. These traits are
   // not affected by access granted by access lists and are obtained prior to granting access list access.
   repeated teleport.trait.v1.Trait original_traits = 5;
+
+  // GitHubIdentity is the external identity attached to this user state.
+  ExternalIdentity git_hub_identity = 6;
```

Comment:
Naming consistency

```suggestion
  // github_identity is the external identity attached to this user state.
  ExternalIdentity github_identity = 6;
```

Reply:
i know this is a weird one. `github_identity` gives `GithubIdentity` in golang,  but `git_hub_identity` gives `GitHubIdentity`. I would prefer `GitHub` than `Github`. 

as for the doc, i am also documenting for the golang side. i don't think we are consistent on that regards. what's your thought on this.

Reply:
Ah, I thought it was a typo. I'm cool with all of this.